std_msgs/Header header
grid_map_msgs/GridMap local_gridmap
nav_msgs/Odometry odom